node/node.go

```diff
@@ -585,10 +585,7 @@ func (node *Node) setupHeartbeatServices(ctx context.Context) error {
 
 func (node *Node) setupMining(ctx context.Context) error {
 	// configure the underlying sector store, defaulting to the non-test version
-	sectorStoreType := proofs.Live
-	if os.Getenv("FIL_USE_SMALL_SECTORS") == "true" {
-		sectorStoreType = proofs.Test
-	}
+	sectorStoreType := proofs.GetSectorStoreType()
```

Comment:
BLOCKING: This should come from the blockchain - specifically from the `StorageMarketActor`. Agree? Disagree?